neurons/config.py

```diff
@@ -136,10 +124,10 @@ def miner_config():
         help="If true, the repo will be made public after uploading.",
     )
     parser.add_argument(
-        "--avg_loss_upload_threshold",
```

Comment:
I agree that this name no longer holds, but miners are used to the old flag name. WDYT about keeping it as-is?